plugins/spu2-x/src/wx/wxConfig.cpp

```diff
@@ -18,471 +18,484 @@
-    dealias_check = new wxCheckBox(this, wxID_ANY, "Use the de-alias filter (Overemphasizes the highs)");
+	numSpeakers = m_audio_select->GetSelection();
 
-    m_mix_box->Add(m_inter_select, wxSizerFlags().Centre());
-    m_mix_box->Add(effect_check, wxSizerFlags().Centre());
-    m_mix_box->Add(dealias_check, wxSizerFlags().Centre());
+	FinalVolume = m_volume_slider->GetValue() / 100.0;
+	SndOutLatencyMS = m_latency_slider->GetValue();
+}
 
-    // Debug Settings
-    debug_check = new wxCheckBox(this, wxID_ANY, "Enable Debug Options");
-    launch_debug_dialog = new wxButton(this, wxID_ANY, "Configure...");
+void MixerTab::Update()
+{
+}
 
-    m_debug_box->Add(debug_check, wxSizerFlags().Expand());
-    m_debug_box->Add(launch_debug_dialog, wxSizerFlags().Expand());
+void MixerTab::CallUpdate(wxCommandEvent& /*event*/)
+{
+	Update();
+}
 
-    // Output Settings
+SyncTab::SyncTab(wxWindow* parent)
+	: wxPanel(parent, wxID_ANY)
+{
+	auto* top_box = new wxBoxSizer(wxVERTICAL);
 
-    // Module
-    m_output_box->Add(new wxStaticText(this, wxID_ANY, "Module"), wxSizerFlags().Centre());
-    m_module.Add("No Sound (Emulate SPU2 only)");
-#ifdef SPU2X_PORTAUDIO
-    m_module.Add("PortAudio (Cross-platform)");
-#endif
-    m_module.Add("SDL Audio (Recommended for PulseAudio)");
-    m_module_select = new wxChoice(this, wxID_ANY, wxDefaultPosition, wxDefaultSize, m_module);
-    m_output_box->Add(m_module_select, wxSizerFlags().Centre());
-#ifdef SPU2X_PORTAUDIO
-    // Portaudio
-    m_portaudio_text = new wxStaticText(this, wxID_ANY, "Portaudio API");
-    m_portaudio_box->Add(m_portaudio_text, wxSizerFlags().Centre());
-#ifdef __linux__
-    m_portaudio.Add("ALSA (recommended)");
-    m_portaudio.Add("OSS (legacy)");
-    m_portaudio.Add("JACK");
-#elif defined(__APPLE__)
-    m_portaudio.Add("CoreAudio");
-#else
-    m_portaudio.Add("OSS");
-#endif
-    m_portaudio_select = new wxChoice(this, wxID_ANY, wxDefaultPosition, wxDefaultSize, m_portaudio);
-    m_portaudio_box->Add(m_portaudio_select, wxSizerFlags().Centre());
-#endif
+	top_box->Add(new wxStaticText(this, wxID_ANY, "Synchronization"), wxSizerFlags().Centre());
+	m_sync.Add("TimeStretch (Recommended)");
+	m_sync.Add("Async Mix (Breaks some games!)");
+	m_sync.Add("None (Audio can skip.)");
+	m_sync_select = new wxChoice(this, wxID_ANY, wxDefaultPosition, wxDefaultSize, m_sync);
 
-    // SDL
-    m_sdl_text = new wxStaticText(this, wxID_ANY, "SDL API");
-    m_sdl_box->Add(m_sdl_text, wxSizerFlags().Centre());
+	auto* adv_box = new wxStaticBoxSizer(wxVERTICAL, this, "Advanced");
 
-    for (int i = 0; i < SDL_GetNumAudioDrivers(); ++i)
-        m_sdl.Add(SDL_GetAudioDriver(i));
+	auto* babble_label = new wxStaticText(this, wxID_ANY, \
+	"For fine-tuning time stretching.\n"\
+	"Larger is better for slowdown, && smaller for speedup (60+ fps).\n"\
+	"All options in microseconds.",\
+	wxDefaultPosition, wxSize(200, 65), wxALIGN_CENTRE_HORIZONTAL);
```

Comment:
It's cutting off the last line 
<img width="352" alt="image" src="https://user-images.githubusercontent.com/3315070/92439459-80566c00-f170-11ea-9dc3-44c7ab95baa5.png">

Could you try this?
```suggestion
	wxDefaultPosition, wxDefaultSize, wxALIGN_CENTRE_HORIZONTAL);
	babble_label->Wrap(300);
```

Reply:
Here's what that winds up looking like on Linux.
![spu2-x5](https://user-images.githubusercontent.com/26599/92445399-269e7380-f169-11ea-8b0c-7e15c34b9fc6.png)
IIRC, this is where I was before I made those changes, with everything centered off to the left for some reason...

Reply:
It stays centered if you remove the `.Expand()` when adding it to the sizer

Reply:
Alright, got it. Committed.